### PR TITLE
fix(completions): add make file completions gen

### PR DESCRIPTION
**Pull request by rubiin**

### CODE REVIEW

**Review by kRHYME7 — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -78,6 +78,9 @@ restore:
 	cp -r $(BACKUPDIR)/confs/* $(DESTDIR)$(etcdir)
 	cp -r $(BACKUPDIR)/extras/* $(DESTDIR)$(usrdir)
 
+completions:
+	complgen aot --bash-script ./shell-completions/Hyde.bash --fish-script ./shell-completions/Hyde.fish --zsh-script ./shell-completions/Hyde.zsh ./shell-completions/Hyde.usage
```
[the remark this answers is not in the document]
Comment (kRHYME7) [reply Makefile:82]: For this is Fine. Thanks!

[the remark this answers is not in the document]
Comment (rubiin) [reply Makefile:82]: maybe we can include something as a pre step that checks all the required deps .

[the remark this answers is not in the document]
Comment (kRHYME7) [reply Makefile:82]: I don't want to add this package as deps but pacman is so great. So we have the 'makedep' where the packages are removed after install.

For manual installation ` make completion  `  we can add the checks later. 

BTW Thanks for your clean MRs we need more contributors like you! It's obvious you got a lot of experience.

**Comment by rubiin** [comment]
Updated to check

**Comment by rubiin** [comment]
Updated to check, seems to be working prompting user to install the package. Also renamed completions to completion

**Comment by kRHYME7** [comment]
Nice Thanks!

**Pull request merged**